modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -43,22 +43,30 @@ class SeqTranslate(site: Site, systems: Systems, settings: Settings) {
   private def dhsFileId(inst: InstrumentSystem): SeqAction[ImageFileId] =
     systems.dhs.createImage(DhsClient.ImageParameters(DhsClient.Permanent, List(inst.contributorName, "dhs-http")))
 
+  private def sendDataStart(obsId: SPObservationID, imageFileId: ImageFileId, dataId: String): SeqAction[Unit] =
+    systems.odb.datasetStart(obsId, dataId, imageFileId).flatMap{
```

Comment:
I think the `if` could be avoided with something like `foldLeftM` but this is low prio